src/main/java/ToDo.java

```diff
@@ -0,0 +1,33 @@
+/**
+ * Represents a task to be done.
+ */
+public class ToDo extends Task {
+    public ToDo(String description) {
+        super(description);
+    }
+
+    public ToDo(String description, boolean isDone) {
+        super(description, isDone);
+    }
+
+    /**
+     * Mark the ToDo task as done.
```

Comment:
Maybe using Marks instead of Mark will be better in following the documentation practices. This is repeated in other comments as well.